src/UI/Styles.cs

```diff
@@ -144,12 +144,15 @@ public static float ScaleFloat(float val)
 			return val * Settings.UIScale * GameSettings.UI_SCALE * GameSettings.UI_SCALE_APPS;
 		}
 
-		// Increasing font size does not affect width as much as height. To avoid excessively wide UIs we scale differently.
-		// Scaling by 80% is simply an emperical factor that is intended to leave the default UI width of 100% scaling intact,
-		// while having a sensible width at high scale value (e.g. 170%).
+		// Increasing font size does not affect width as much as height. To avoid excessively wide UIs we scale with UIPanelWidthSScale rather than	UIScale
 		public static float ScaleWidthFloat(float val)
 		{
-			return val * (1.0f + (Settings.UIScale * GameSettings.UI_SCALE * GameSettings.UI_SCALE_APPS - 1.0f) * 0.8f);
+			return val * (1.0f + (Settings.UIPanelWidthScale * GameSettings.UI_SCALE * GameSettings.UI_SCALE_APPS - 1.0f));
```

Comment:
you simplify this by removing the 1.0f values, they don't do anything anymore

Reply:
thanks ;)